addon/templates/components/frost-expand.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="frost-expand-label"  data-test={{hook (concat hook '-label')}} onClick={{action '_onClick'}}>
+  {{frost-icon
+    class='frost-expand-label-icon'
```

Comment:
`frost-icons` have an automagical class created based on the pack and icon

`frost-icon-<pack>-<icon>` so this instance should already have a `frost-icon-frost-expand-collapse` class available

Also, any reason we aren't re-using `chevron` here?